jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -303,6 +303,7 @@ pipeline {
                                         testType: "integ-test",
                                         componentName: "${COMPONENT_NAME}",
                                     )
+                                    echo("Download report manifest from https://ci.opensearch.org/ci/dbc/integ-test/${env.version}/${env.buildId}/${env.platform}/${env.architecture}/${env.distribution}/test-results/${env.BUILD_NUMBER}/integ-test/test-report.yml")
```

Comment:
How about attach the test-report manifest as an artifact like we used to for buildInfo.yml?
I believe that would be helpful than an echo statement. WDYT?

Reply:
@gaiksaya Yes! I was also thinking about this before. Maybe we can add both: echo with the URL link and also use `archiveArtifacts artifacts:` to attach on each jenkins job.

Reply:
Sure! When you view in blue ocean view if it is success no one goes and checks the stage also its difficult to parse through the logs to get the URL. Hence though instead of this directly attach the file to the job.

Reply:
It's added now in the latest commit. Please help review it again. Thanks.